systems/plants/joints/FixedJoint.h

```diff
@@ -25,7 +27,7 @@ class DLLEXPORT_DRAKEJOINT FixedJoint: public DrakeJoint
       Gradient<Vector6d, Eigen::Dynamic>::type* dmotion_subspace_dot_times_vdq = nullptr,
       Gradient<Vector6d, Eigen::Dynamic>::type* dmotion_subspace_dot_times_vdv = nullptr) const;
 
-  virtual void randomConfiguration(Eigen::Ref<Eigen::VectorXd>& q, std::default_random_engine& generator) const;
+  virtual Eigen::VectorXd randomConfiguration(std::default_random_engine& generator) const;
```

Comment:
We discussed this and determined that in this case it's actually required since the base class method is pure virtual.